core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -103,7 +103,7 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
       val taskSortColumn = Option(parameterTaskSortColumn).map { sortColumn =>
         UIUtils.decodeURLParameter(sortColumn)
       }.getOrElse("Index")
-      val taskSortDesc = Option(parameterTaskSortDesc).map(_.toBoolean).getOrElse(false)
+      val taskSortDesc = Option(parameterTaskSortDesc).exists(_.toBoolean)
```

Comment:
Still not sure about whether this is clearer. I'm neutral. @HyukjinKwon is right about `.contains` I think so I'd double check this existed in 2.10?